src/cmd-generate-release-meta

```diff
@@ -35,13 +36,9 @@ parser = ArgumentParser()
 parser.add_argument("--workdir", help="cosa workdir")
 parser.add_argument("--build-id", help="build id")
 parser.add_argument("--output", help="Output to file; default is build directory")
+parser.add_argument("url", metavar='URL', help="URL to a coreos-assembler meta.json", default=[], nargs='*')
```

Comment:
Do we actually need this? Can't we just have the pipeline do `cosa buildprep` first? That seems more in line with the rest of cosa.

Reply:
It's *so* convenient though - e.g. when working on patches to openshift/installer I just ran this command locally to generate the stream json to commit to that repo (because we haven't deployed this yet).
We don't need the ostree tarball for this either.

Reply:
`cosa buildprep` by default doesn't fetch the OSTree tarball, so it's a fairly lightweight operation.

Hmm, I think it's the "positionalness" of the argument that seems out of place to me. WDYT about making it a repeatable `--meta-json` instead?

Related: for hacking purposes, might as well have the URI support local paths to `meta.json`s too?

Reply:
Switched to `--url`.

> Related: for hacking purposes, might as well have the URI support local paths to meta.jsons too?

When would that happen?  Feels like you either have a local cosa build or a remote URL?